README.md

```diff
@@ -1 +1,86 @@
-Website for Raven-OS
+# Website's API
```

Comment:
Title and first line should be updated to talk about the whole website, and not only it's backend.